### ISSUE TITLE
Building an Excel DNA addin doesn't find app.config when added to a project as a link

### ISSUE THREAD

**Issue by chrish29** [Excel-DNA/ExcelDna#177]
Hi, 

I believe I have come across an issue when building an ExcelDNA AddIn with version 0.34.6. 

The build tasks run to create the XLL but it does not create the XLL.config if the project's 'app.config' was added as a linked file.

Please see the sample linked below. The project named 'LinkedAppConfig' does not output the xll.config file when built. 
[ExcelDNALinkedAppConfig.zip](https://github.com/Excel-DNA/ExcelDna/files/1920001/ExcelDNALinkedAppConfig.zip)

Many Thanks,
Chris

**Comment by govert** [comment]
@chrish29 This might have changed in version 0.34 since the build process and related tasks got updated, including dealing with the .config files explicitly. I didn't go back to look, but confirmed that this is an issue with the current version.

I've now pushed a commit that adds support for linked app.config files (971d9f96af3b80ee54a5a9ad7ac4cce0d0087c2a), and it should be working with the next test version (after version 0.35.1).
